variables.tf

```diff
@@ -11,7 +11,7 @@ variable "name" {
 }
 
 variable "image" {
-  default = "alpine"
+  default = ""
```

Comment:
Why no default image?

Reply:
This is not a generic `Docker` image from `Docker Hub`.
This is a specific `CodeBuild` image created by AWS to build a specific environment, e.g. Docker, Node, Java, etc. 
Example would be `aws/codebuild/docker:1.12.1` or `aws/codebuild/eb-nodejs-6.10.0-amazonlinux-64:4.0.0`.
http://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref.html
`Alpine` is not one of them.
We should remove the `default` completely since it's specific to the environment we build.